docs/exchange-operators/rosetta/samples/requests.mdx

```diff
@@ -34,13 +56,49 @@ def _request(path, data=None):
     return r.json()
 ```
 
+</TabItem>
+</Tabs>
+
 ### Block, Transaction, Account Identifiers and the Pubkey object
 
 According to Rosetta request and response models, each block, transaction and account are identified by an identifier object.
 A pubkey is also represented by special object.
 
 To implement helpers to construct the objects:
 
+<Tabs>
+<TabItem value="ts" label="TypeScript">
+
+```ts
+function makeBlockIdentifier(idOrHash: number | string) {
+    const identifierKey = (typeof idOrHash === "number") ? "index" : "hash"
+    return {
+        block_identifier: {[identifierKey]: idOrHash}
+    }
+}
+
+function makeTxIdentifier(hash: string) {
```

Comment:
I suggest cleaning this cope snippet up a little bit

Reply:
I'd suggest addressing these comments as a separate PR once main has been merged